spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -7,62 +7,156 @@
 
   describe 'GET#index' do
     context 'given a logged in user' do
-      context 'given the user has keywords' do
-        it 'returns success status' do
-          user = Fabricate(:user)
-          Fabricate.times(3, :keyword, user: user)
+      context 'given the user retrives all of their keywords' do
+        context 'given the user has keywords' do
+          it 'returns success status' do
+            user = Fabricate(:user)
+            Fabricate.times(3, :keyword, user: user)
 
-          get api_v1_keywords_path, headers: create_token_header(user)
+            get api_v1_keywords_path, headers: create_token_header(user)
 
-          expect(response).to have_http_status(:success)
-        end
+            expect(response).to have_http_status(:success)
+          end
+
+          it 'returns three items' do
```

Comment:
```suggestion
          it 'returns 3 items' do
```

As in the tests above, we use digits instead of full letters numbers 😇